nemo/nemo/backends/pytorch/module_wrapper.py

```
@@ -10,13 +10,6 @@ class TrainableNeuralModuleWrapper(NeuralModule, nn.Module):
     """This class wraps an instance of Pytorch's nn.Module and
     returns NeuralModule's instance."""
 
-    @staticmethod
```

Can we throw a deprecation error instead of removing this function?